docs/tutorials/conversion/import_saved_model.md

```diff
@@ -2,9 +2,7 @@
 
 TensorFlow GraphDef based models (typically created via the Python API) may be saved in one of following formats:
 1. TensorFlow [SavedModel](https://www.tensorflow.org/programmers_guide/saved_model#overview_of_saving_and_restoring_models)
-2. [Frozen Model](https://www.tensorflow.org/mobile/prepare_models#how_do_you_get_a_model_you_can_use_on_mobile)
```

Comment:
frozen model is still supported.

Reply:
Here is a link that explain how to freeze a graph https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/r1/convert/python_api.md#checkpoints

Use it for Frozen model instead.